Automattic/jetpack
# Social Icons Widget: Adds Patreon social icon for social icons widget and menu

Fixes #14122

#### Changes proposed in this Pull Request:

* Adds the Patreon social icon to both the social icons widget and social menu (for supported themes).

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


Nope, don't think so.

#### Testing instructions:






Testing the Social Icons Widget:

* Open the customizer
* Add a social icons widget 
* Add a Patreon URL, e.g. https://patreon.com
* You should see the Patreon logo appear in the widget area

![patreon widget follow us](https://user-images.githubusercontent.com/33553323/105077262-410f2480-5a5a-11eb-9dd3-5dcefbdbaf84.png)


Testing the Social Menu

* Activate a theme that supports Jetpack social icons (e.g. Dara).
* Open the customizer and add/edit the social menu.
* Add a Patreon URL as a custom link, e.g. https://patreon.com.
* You should see the Patreon logo appear in the social media menu.

![patreon social menu](https://user-images.githubusercontent.com/33553323/105077256-3ce30700-5a5a-11eb-97e3-7ae975f4d4e2.png)


#### Proposed changelog entry for your changes:


* Social Icons Widget: Adds Patreon icon for social icons widget and social menus utilizing Jetpack's social icons.
